guides/environments.md

```diff
@@ -0,0 +1,29 @@
+# Environments
+
+Every error reported to Rollbar is required to have an `environment` property.  This value can contain any non-empty alphanumeric string of up to 255 characters, so it can be used in a number of ways.
```

Comment:
- Technically it's not actually required. If you omit environment, it'll become the string `"unknown"`
- Can contain "any non-empty string of up to 255 latin-1 characters"